### DISCUSSION TITLE
TableVirtuoso + Drag&Drop

### DISCUSSION THREAD

**Discussion by AliceQD** [petyosi/react-virtuoso#575] [Q&A]
I've been attempting to add Drag & Drop functionality to my table (which was created using the VirtuosoTable component), but I'm having trouble doing so.
I've been following these examples: [example1](https://codesandbox.io/s/e6vmq?file=/src/index.js:1742-1762), [example2](https://codesandbox.io/s/lxvof?file=/App.js), which are both excellent, but only illustrate how to do it with regular lists.
Is it even feasible to add a functionality like this using VirtuosoTable?

I'd appreciate any assistance or, if possible, an example on this one.

This library is fantastic, and I appreciate all of your hard work. (I don't usually write issues, so I am sorry if I sounded mean or anything like that)

**Comment by nick-catalist** [comment]
I have been trying to get TableVirtuoso to work with https://github.com/hello-pangea/dnd for two days and I'm absolutely losing my mind...

**Reply by xmsz-stu** [reply]
"""
Is it functioning?
"""
